### ISSUE TITLE
DWARF2 warning in arch/mips/vdso/elf.S

### ISSUE THREAD

**Issue by nathanchance** [ClangBuiltLinux/linux#607]
```
arch/mips/vdso/elf.S:14:1: warning: DWARF2 only supports one section per compilation unit
.pushsection .note.Linux, "a",@note ; .balign 4 ; .long 2f - 1f ; .long 4484f - 3f ; .long 0 ; 1:.asciz "Linux" ; 2:.balign 4 ; 3:
^
arch/mips/vdso/elf.S:34:2: warning: DWARF2 only supports one section per compilation unit
 .section .mips_abiflags, "a"
 ^
```

**Comment by nickdesaulniers** [comment]
We used to get these in x86 IIRC, not sure what we did to fix that there. Maybe @mkaehlcke remembers?

**Comment by nickdesaulniers** [comment]
dwarf2 can be deleted from the kernel once the minimum supported version of GCC moved past 4.6.

**Comment by nickdesaulniers** [comment]
I made #716 non-arch-specific, then closed #919 as a dupe of it, so let's do that here, too.
